chia/rpc/wallet_rpc_api.py

```diff
@@ -4180,6 +4180,7 @@ async def dl_update_multiple(
                     bytes32.from_hexstr(launcher),
                     bytes32.from_hexstr(root),
                     tx_config,
+                    fee=uint64(request.get("fee", 0)),
```

Comment:
Just to make sure I'm reading this correctly, this is correcting that we missed this in the existing implementation?  It seems relatively unrelated.

Reply:
yes, the original `dl_update_multiple` didn't take the `fee` param as far as I can tell. I added it to glue it with our CLI interfaces. note, for changing `t` trees, and `m` mojos, we actually spend `m * t` mojos, not just `m`. it's worth discussing too if we want this behaviour, or spend `m // t` per transaction.

Reply:
Alrighty, thanks.  I guess I'll leave this open for the moment due to your suggestion of discussing the fees.  Once that is tracked somewhere else or discussed and concluded then this conversation can be resolved.